src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -155,8 +159,7 @@ private OpenApiSchema GeneratePrimitiveSchema(DataContract dataContract)
 
             if (dataContract.EnumValues != null)
             {
-                schema.Enum = dataContract.EnumValues
-                    .Distinct()
+                schema.Enum = dataContract.EnumValues.Values
```

Comment:
Removed `Distinct()` here, as it's theoretically done already (see comment below). However, with pathalogical enum member name remapping, the values here could be distinct even if though the keys can't be. There isn't a test case that makes this happen, currently, and not really sure if it needs addressing?

Actually, on reflection, maybe this is exactly the situation that the previous enum duplicates test case was really intended to simulate? In which case I guess it does need addressing, perhaps with a quick test which actually uses enum member renaming to get to this situation(?), and by restoring `Distinct()` here.